packages/block-editor/src/utils/index.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Internal dependencies
+ */
+export { default as transformStyles } from './transform-styles';
```

Comment:
This is looking good, my last question is whether we should publish this as is or make it experimental? I'm on the fence here as it seems to be a pretty stable API. Thoughts?

Also do we consider a change in the way we generate those styles as a breaking change? Say we automatically add a "width" to all blocks or something? (if it's the case I think it should be experimental or unstable)
 
cc @aduth @jorgefilipecosta @jasmussen

Reply:
It seems like if you're on the fence, maybe keep it unstable for a release, then revisit?

Reply:
I agree with the idea of experimental for a release.

Reply:
> I agree with the idea of experimental for a release.

Will you plan to follow-up on this, or should we create an issue to track it?

Reply:
Hi @aduth, there is no need to create an issue, I will create a PR very soon, given that the experimental status should be removed in the next release.